utils/shake.ts

```diff
@@ -0,0 +1,36 @@
+import { Accelerometer, AccelerometerMeasurement } from "expo-sensors";
+import { useState, useEffect } from "react";
+
+const THRESHOLD = 4; // Adjust this value based on your sensitivity preference
```

Comment:
In testing on Android this feels pretty hard to get the dev menu to open also I'm seeing some pretty bad Android performance on this branch comparing to main